src/main/java/lotto/manageError/CheckError.java

```diff
@@ -0,0 +1,49 @@
+package lotto.manageError;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CheckError {
+ public static void dividedThousand(int price){
+   if (price % 1000 != 0){
+   System.out.println("[Error] 남은 동전이 없어 잔돈을 드리지 못 합니다. 다시 입력해 주세요.");
+   throw new IllegalArgumentException();
```

Comment:
예외 메시지를 한번에 처리하는 것이 좋습니다.
```suggestion
   throw new IllegalArgumentException("[Error] 남은 동전이 없어 잔돈을 드리지 못 합니다. 다시 입력해 주세요.");
```